packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -279,6 +279,20 @@ export interface BucketAttributes {
   readonly bucketWebsiteNewUrlFormat?: boolean;
 
   readonly encryptionKey?: kms.IKey;
+
+  /**
+   * The account this existing bucket belongs to.
+   *
+   * @default - it's assumed the bucket belongs to the same account as the scope it's being imported into
+   */
+  readonly account?: string;
+
+  /**
+   * The region this existing bucket is in.
+   *
+   * @default - it's assumed the bucket is in the same region as the scope it's being imported into
+   */
+  readonly region?: string;
```

Comment:
Is the `region` even relevant for the purposes of this PR?

Reply:
Yep! Used in the CodePipeline construct.